env.example

```diff
@@ -129,6 +129,9 @@ ETHERPAD_SKIN_VARIANTS="super-light-toolbar super-light-editor light-background
 # SIP server transport
 #JIGASI_SIP_TRANSPORT=UDP
 
+# SIP default room name for incoming calls lacking of Invite header
+#JIGASI_SIP_DEFAULT_ROOM=conference
```

Comment:
Please drop this part, I want to move all the examples to the handbook instead. Otherwise LGTM.

Reply:
I see. It's done.